packages/step-vehicle-inspection/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@raincatcher/vehicle-inspection",
+  "name": "@raincatcher-demo/demo-step-vehicle-inspection",
```

Comment:
is this the right name here? looks like it's referenced by `@raincatcher-demo/vehicle-inspection` in other places